packages/agent-toolkit/src/core/tools/platform-api-tools/get-instructions-to-create-live-workflow-tool.ts

```diff
@@ -0,0 +1,284 @@
+  name = 'get_instructions_to_create_live_workflow';
+  type = ToolType.READ;
+
+  getDescription(): string {
+    return `When the user asks to create a workflow, you must use this tool, which provides instructions on how to create a workflow.
+    This tool is very important and should be used only when the user asks to create a workflow.
+    In general, create live workflows when the user asks you to do something automatically on monday.com items or boards.
+    For example: when an item is created, set the status to done`;
+  } 
+
+  getInputSchema(): Record<string, never> {
+    return {};
+  }
+
+  async execute(): Promise<ToolOutputType<never>> {
+    const instructions = `
+# Instructions to Create a Live Workflow
+
+## General Explanation:
+- A workflow is a set of blocks that are connected to each other (trigger → action → action → ...).
+- A block is a step in the workflow; it can be a trigger or an action. It has input fields and output fields.
+- A workflow block is a configuration of a block. It has a block reference ID, input fields, and output fields.
+- A workflow variable is a variable that is used in the workflow. It has a unique key, a value, and dependencies.
+- To fill the value of a variable, you need to use the remote options query.
+- Workflow host data contains the data of the workflow's host. It has an ID and a type.
+
+To create a live workflow in monday.com, follow these steps:
+
+1. **Fetch the blocks including the input fields config:**
+   Each block represents an action or a trigger. Some blocks have input fields and some have output fields.
+   In general, you can understand what a block does by its description or name.
+   The "kind" field in the block represents the type of the block (trigger or action).
+
+   'Custom input fields' represent parameters that are declared as field type app features. Their identifier is the 'fieldTypeReferenceId' (or 'id' in the fieldTypeData).
+   There are dependencies that tell us what values we need to know to fetch the options for the custom input field's value.
+   For example, if the custom input field is a status column, the dependency is the board ID.
+
+   **Example query:**
+query {
+  blocks {
+    blocks {
+      id
+      description
+      name
+      kind
+      inputFieldsConfig {
+        fieldTitle
+        fieldKey
+        ... on CustomInputFieldConfig {
+          fieldTypeReferenceId
+          fieldTypeData {
+            id
+            dependencyConfig{
+              optionalFields{
+                sourceFieldTypeReferenceId
+                targetFieldKey
+              }
+              orderedMandatoryFields{
+                sourceFieldTypeReferenceId
+                targetFieldKey
+              }
+              
+            }
+          }
+        }
+        ... on PrimitiveInputFieldConfig {
+          primitiveType
+        }
+        ... on InterfaceInputFieldConfig{
```

Comment:
does it make sense to query the interface? thought it's just implemention details and we always fetch a specific implementation